test/jest_integration/saml_auth.test.ts

```diff
@@ -284,87 +284,87 @@ describe('start OpenSearch Dashboards server', () => {
     await driver.quit();
   });
 
-  it('Tenancy persisted after Logout in SAML', async () => {
-    const driver = getDriver(browser, options).build();
```

Comment:
After discussion with @devardee, we concluded that this test is flaky and needs to be fixed. I will open a github issue soon for this test.
cc: @anijain-Amazon @prabhat-chaturvedi